src/react-apps/applications/runtime/src/types/global.ts

```diff
@@ -236,4 +236,26 @@ declare global {
   export interface IValidationResults {
     [id: string]: IComponentValidations;
   }
+  export interface IAttachment {
+    uploaded: boolean;
+    deleting: boolean;
+    name: string;
+    size: number;
+    id: string;
+  }
+
+  export interface IAttachments {
+    [attachmentType: string]: IAttachment[];
```

Comment:
What can the attachmentType key be? Is it the memeType?

Reply:
No, its more of a category. It could for instance be 'Catpictures'. So a string describing the group of attachments. For now it's just a guid, but issue #1137 looks into this.